admin.php

```diff
@@ -7,39 +7,124 @@
  * @author Dominik Brader <support@findologic.com>
  */
 
-// must be run within Dokuwiki
-if(!defined('DOKU_INC')) {
-    define('DOKU_INC',realpath(dirname(__FILE__).'/../../../').'/');
+if (!defined('DOKU_INC')) {
+    die('Must be run within Dokuwiki!');
 }
 
-class admin_plugin_findologicxmlexport extends DokuWiki_Admin_Plugin {
+require_once(__DIR__ . '/vendor/autoload.php');
+require_once(__DIR__ . '/PageGetter.php');
+
+class admin_plugin_findologicxmlexport extends DokuWiki_Admin_Plugin
```

Comment:
Conventially class names start with a capital letter. Ideally they should use caps case, but Dokuwiki apparently uses some weird underscore caps case so you should stick to that.

Reply:
DokuWiki weird Approved &copy;